README.md

```diff
@@ -101,15 +101,17 @@ First [Build Gaia](#build-gaia) and [Download the testnet configurations](#downl
 Here's an example build command:
 
 ```bash
-yarn run build --commit=HEAD --network=gaia-7001
+yarn run build --commit=HEAD --network=gaia-7005
 ```
 
 You can specify `--help` to see all options with explanations.
 
-Run the app.
+To unzip the app:
 
 ```bash
-open builds/Cosmos-{platform}-x64/Cosmos.app
+open builds/Voyager/Cosmos_Voyager-v0.9.4-Linux.tar.gz
+open builds/Voyager/Cosmos_Voyager-v0.9.4-macOS.zip
+open builds/Voyager/Cosmos_Voyager-v0.9.4-Windows.zip
```

Comment:
Is this tested?

Reply:
not sure what you mean. if the user is in the correct directory it should open and unzip the files.

`open builds/Cosmos-{platform}-x64/Cosmos.app` doesn't work for any platform because we put builds in `/Voyager` now.

is there something i need to be looking out for that i'm not thinking of?

Reply:
There is no `open` cmd on Windows/Linux ;)

Reply:
I added a proposal, let me know what you think.